ca/server.go

```diff
@@ -90,6 +93,13 @@ func NewServer(store *store.MemoryStore, securityConfig *SecurityConfig, rootCAP
 	}
 }
 
+// ExternalCA returns the current external CA
+func (s *Server) ExternalCA() *ExternalCA {
```

Comment:
Nope, just the test.  I can make the server test part of the `ca` package instead of the `ca_test` package to access it directly, if you prefer.